### DISCUSSION TITLE
Export data to schematic

### DISCUSSION THREAD

**Discussion by Dmistass** [MARSTeamMC/Image2Banners#3] [Ideas]
If the picture is not small, it will be easier to export the schematic right away to easily insert it into the world. This would be a great addition.
I think it's possible to use the python "mcschematic" module here.

**Comment by MDen-MT** [comment]
Sounds like a great idea! I’m going to look into that for v1.2.0!

**Comment by MDen-MT** [comment]
Export through a structure block was added in v2.0.0, no need in export to schematic.
